source/deployment/bots.rst

```diff
@@ -0,0 +1,47 @@
+Bots
+============================================
+
+Our `Integrations Directory <https://integrations.mattermost.com>`_ has several integrations to connect bots in Mattermost.
+
+Here are some popular options below, including self-hosted on-prem or self-hosted private cloud solutions, and vendor-hosted SaaS solutions.
+
+WelcomeBot
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+ - Welcome users to your Mattermost instance, developed by Mattermost.
+ - Add a Welcome Bot that helps add new team members to channels to improve onboarding and HR processes.
+ - Source code + docs: https://github.com/mattermost/mattermost-plugin-welcomebot
+
+BotKube
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+ - Monitor and debug Kubernetes resources, developed by InfraCloud Technologies.
+ - Helps you monitor your Kubernetes cluster, debug critical deployments and get recommendations for standard practices.
+ - Source code + docs: https://github.com/infracloudio/botkube
+
+Sample Golang bot
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+ - Sample bot for Go driver, developed by Mattermost.
+ - Learn how to use the Mattermost Go driver to interact with a Mattermost server, listen to events and respond to messages.
+ - Use API for simple tasks such as logging in to your server, creating a channel and posting a message.
+ - Source code + docs: https://github.com/mattermost/mattermost-bot-sample-golang
+
+Hubot Adapter
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+ - Hubot adapter for Mattermost written in Javascript that uses Web API and Websockets, developed by Andy Lo-A-Foe.
+ - Use the bot to listen for commands and to execute actions based on your requests.
+ - Invite your bot to any Mattermost channel just like a regular user.
+ - Source code + docs: https://github.com/loafoe/hubot-matteruser
+
+Python Chatbot
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+ - A python based chat bot for Mattermost, developed by Alex Tzonkov.
+ - **XXXXXX**
```

Comment:
@aaronrothschild Wondering if you can help with more details on this bot as I don't quite understand the features.